src/Microsoft.Maui.Graphics/Color.cs

```diff
@@ -1,430 +1,461 @@
 using System;
-using System.Diagnostics;
-using System.Globalization;
-
-namespace Microsoft.Maui.Graphics
-{
-	[DebuggerDisplay("Red={Red}, Green={Green}, Blue={Blue}, Alpha={Alpha}")]
-	public class Color
-	{
-		public readonly float Red;
-		public readonly float Green;
-		public readonly float Blue;
-		public readonly float Alpha = 1;
-
-		public Color(float gray)
-		{
-			Red = Green = Blue = gray.Clamp(0,1);
-		}
-
-		public Color(float red, float green, float blue)
-		{
-			Red = red.Clamp(0, 1);
-			Green = green.Clamp(0, 1);
-			Blue = blue.Clamp(0, 1);
-		}
-
-		public Color(float red, float green, float blue, float alpha)
-		{
-			Red = red.Clamp(0,1);
-			Green = green.Clamp(0,1);
-			Blue = blue.Clamp(0,1);
-			Alpha = alpha.Clamp(0,1);
-		}
-
-		public Color(string colorAsHex)
```

Comment:
Removed this in favor of FromArgb and FromRgba as this is already causing confusion in the codebases because Forms had the format ARGB (like CSS) and the format here of RGBA is going to be confusing to users.